bitcraze_crazyflie_2/cf2.xml

```diff
@@ -0,0 +1,126 @@
+<mujoco model="cf2">
+  <option integrator="RK4" density="1.225" viscosity="1.8e-5"/>
+
+  <compiler inertiafromgeom="false" meshdir="cf2" autolimits="true"/>
+
+  <default>
+    <default class="visual">
+      <geom group="2" type="mesh" contype="0" conaffinity="0"/>
+    </default>
+    <default class="collision">
+      <geom group="3" type="mesh"/>
+    </default>
+  </default>
+
+  <asset>
+    <material name="polished_plastic1_1" specular="0.5" shininess="0.225" rgba="0.631373 0.658824 0.678431 1.000000"/>
```

Comment:
* Can you remove the suffixes (e.g. `_1`) from the material names?
* Can you change "Material15_001" to the actual color?